index.bs

```diff
@@ -5796,6 +5800,41 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 
 For the purposes of user agent automation and [=web application=] testing, this document defines a number of [[WebDriver]] [=extension commands=].
 
+## WebAuthn WebDriver Capability ## {#sctn-automation-webdriver-capability}
+
+In order to advertise the availability of the [=extension commands=] defined below, a new [=extension capability=] is added.
+
+<figure id="table-virtualAuthenticatorsWebdriverCapability" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>Capability</th>
+                <th>Key</th>
+                <th>Value Type</th>
+                <th>Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Virtual Authenticators Support</td>
+                <td>`":virtualAuthenticators"`</td>
```

Comment:
the dfn of [extension capabilities](https://w3c.github.io/webdriver/#dfn-extension-capability) says "Extension capabilities’ key must contain a ":" (colon) character, denoting an implementation specific namespace. ... As with extension commands, it is suggested that the key used to denote the extension capability namespace is based on the vendor keywords listed in [CSS21] and precedes the first ":" character in the string."

Ought we define such a extension capability namespace, rather than leaving it null as in `:virtualAuthenticators` ?  I do not know whether doing so is a "must" or not.  If so, then perhaps `WebAuthn:virtualAuthenticators` ?

Reply:
Considering this might be the first spec other than webdriver itself to add an extension capability, this is a good idea. Renamed the capability `webauthn:virtualAuthenticators`.